nessie/merged/sis_profile.py

```diff
@@ -78,17 +78,7 @@ def merge_holds(sis_profile_feed, sis_profile):
 
 
 def merge_sis_profile_academic_status(sis_profile_feed, sis_profile):
-    # It is possible to receive multiple academic statuses. We'll prefer an undergraduate enrollment, but
-    # otherwise select the first well-formed status that is not a Law enrollment.
-    academic_status = None
-    for status in sis_profile_feed.get('academicStatuses', []):
-        status_code = status.get('studentCareer', {}).get('academicCareer', {}).get('code')
-        if status_code and status_code == 'UGRD':
-            academic_status = status
-            break
-        elif status_code in {'UCBX', 'GRAD'}:
-            academic_status = status
-            next
```

Comment:
In the EDL world we filter on academic careers before constructing the feed, so we'll never get an `academicStatuses` field with multiple statuses.